blog/2023/10-01-a-call-for-developers/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'A Call for Developers'
+description: "Jellyfin needs your help to get better, so join us this October and beyond!"
+authors: joshuaboniface
+date: 2023-10-01
+slug: a-call-for-developers
+tags: [project]
+---
+
+![I want YOU to contribute to Jellyfin](./contribute-to-jellyfin.jpg)
+
+This year, DigitalOcean's [Hacktoberfest](https://hacktoberfest.org) is in it's 10th year. In previous years, we haven't made too much of it, aside from adding labels to our subprojects to allow contributors to count their contributions towards rewards. While those are a little less impressive this year, within our project, this is our most important year yet.
+
+We'd like to call on you, the wider community, to help make Jellyfin better! We need contributors, fresh ideas and blood to help the project move past our current funk and into something more.
+
+Will you join us?
+
+<!--truncate-->
+
+We keep a close eye on the buzz about Jellyfin around Reddit and the Internet more broadly, even if we've closed our own subreddit. The last couple weeks have seen a few major Plex-related kerfufles and the comments inevitably include the following exchange:
+
+    Person A: Use Jellyfin instead
+    Person B: Jellyfin sucks, it lacks client X or feature Y
+    Person C: Works for me
+    Person D: It needs polish and more apps
+
+Sometimes one of us will jump in, but I think it's high time for us to make a public request for something that's been on our minds as a team for quite a while.
+
+For those who are not aware, Jellyfin is a bit unusual, even within the FLOSS world. We are *explicitly anti-commercial*. Our team is entirely composed of volunteers who contribute because they like what we're doing and can help out in whatever ways they can. This includes developers, translators, and community members who help moderate the community and assist others in troubleshooting and working through problems.
+
+No one gets paid for Jellyfin work, by design. We're not backed by any company or "organization" with its own agenda. We have no "monetization plan" or anything of the sort, and actively avoid bug bounties. Jellyfin isn't a "product" in the commercial sense of the word. And while we do take donations, these *only* cover our infrastructure costs and the rare piece of developer client hardware as needed; we don't pay developers out of donations, and never will. Mmoney is not an issue (for what we use it for), and that's not really what we need. This is a choice we've made from day 1 to avoid Jellyfin following in the footsteps of nearly every other player in this space, who start out as FLOSS, start adding user-hostile features in the name of money, and then go closed source. We want to be different. Want to be volunteer-only and working on this thing to help make media streaming available to everyone.
```

Comment:
```suggestion
No one gets paid for Jellyfin work, by design. We're not backed by any company or "organization" with its own agenda. We have no "monetization plan" or anything of the sort, and actively avoid bug bounties. Jellyfin isn't a "product" in the commercial sense of the word. And while we do take donations, these *only* cover our infrastructure costs and the rare piece of developer client hardware as needed; we don't pay developers out of donations, and never will. Money is not an issue (for what we use it for), and that's not really what we need. This is a choice we've made from day 1 to avoid Jellyfin following in the footsteps of nearly every other player in this space, who start out as FLOSS, start adding user-hostile features in the name of money, and then go closed source. We want to be different. Want to be volunteer-only and working on this thing to help make media streaming available to everyone.
```